.github/workflows/bot-staging-explorer-eth.yml

```diff
@@ -37,5 +37,5 @@ jobs:
           SLACK_API_TOKEN: ${{secrets.SLACK_API_TOKEN}}
           SLACK_ICON_EMOJI: ":bot-seed2:"
           SLACK_CHANNEL: explorer-bot-stg
-          BOT_FILTER_CURRENCIES: avalanche_c_chain,bsc,polygon,ethereum,ethereum_classic,ethereum_goerli
+          BOT_FILTER_CURRENCIES: avalanche_c_chain,bsc,polygon,ethereum,ethereum_classic,ethereum_goerli,ethereum_sepolia,ethereum_holesky
```

Comment:
Please make this work on either nitrogen or oxygen

Reply:
We chose to make it run daily, no update needed

Reply:
To be consistent with what is done for the goerli testnet today, there is nothing to update, which mean that these 2 new testnets will be used for daily nonreg with Nitrogen, as it's already the case today for goerli 👍